modules/cli/src/main/scala/coursier/cli/options/CacheOptions.scala

```diff
@@ -15,58 +15,62 @@ import scala.concurrent.duration.Duration
 // format: off
 final case class CacheOptions(
 
-  @Group("Cache")
+  @Group(OptionGroup.cache)
   @Help("Cache directory (defaults to environment variable COURSIER_CACHE, or ~/.cache/coursier/v1 on Linux and ~/Library/Caches/Coursier/v1 on Mac)")
     cache: Option[String] = None,
 
-  @Group("Cache")
+  @Group(OptionGroup.cache)
+  @Hidden
   @Help("Download mode (default: missing, that is fetch things missing from cache)")
   @Value("offline|update-changing|update|missing|force")
   @Short("m")
     mode: String = "",
 
-  @Group("Cache")
+  @Group(OptionGroup.cache)
+  @Hidden
```

Comment:
I wouldn't hide this one. It can be useful, even for non-advanced users.

Reply:
Done